使用环境变量 GCQ_LEVELDB=true 启动时报错

<!--

！请不要删除此处内容！

在您发布此Issue前, 请您花一点时间查看下面几条指引🔽

1: ❗ | 确定没有相同问题的ISSUE已被提出. (教程: https://github.com/Mrs4s/go-cqhttp/issues/633)
2: 🌎| 请准确填写环境信息.
3: ❔ | 打开DEBUG模式复现，并提供出现问题前后至少 10 秒的完整日志内容。请自行删除日志内存在的个人信息及敏感内容。
4: ⚠ | 如果涉及内存泄漏/CPU占用异常请打开DEBUG模式并下载pprof性能分析.

注: 如果您不知道如何有效、精准地表述，我们建议您先阅读《提问的智慧》
(链接: https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

请确保您已经仔细阅读此教程，并勾选下方的确认框。(将 [ ] 修改为 [x])
--------
- [x] 我已经仔细阅读上述教程和"提问前需知 [图+文]": https://github.com/Mrs4s/go-cqhttp/issues/633
- [x] 我已知晓并同意，如果我不遵循以下格式提交 Issue，或者我使用的并非最新版本，或者我没有提供足够的环境信息，则我的 Issue 可能会被无条件自动关闭和锁定。
- [x] 我已知晓并同意，我仅需要把选项前的 [ ] 替换为 [x]。如果我删除、修改这些复选框的其他部分，或是在 x 之前或之后留了空格，则我的 Issue 可能会被无条件自动关闭和锁定。
- [x] 我已知晓并同意，此处仅用于汇报程序中存在的问题。若这个 Issue 是关于其他非程序本身问题或是新功能需求，则我的 Issue 可能会被无条件自动关闭和锁定。（这些问题应当在 Discussion 板块提出。）
--------
-->

**环境信息**
go-cqhttp版本: 1.0.0-b6
运行环境: linux
连接方式: ws
使用协议: ws

**在最新的release版本中能否复现**
能

**bug内容**
使用环境变量 GCQ_LEVELDB=true 启动时报错

**复现方法**
```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/Mrs4s/go-cqhttp/global/config.Get.func1()
        github.com/Mrs4s/go-cqhttp/global/config/config.go:178 +0x645
sync.(*Once).doSlow(0x159bbc0, 0xedf0b0)
        sync/once.go:68 +0xec
sync.(*Once).Do(...)
        sync/once.go:59
github.com/Mrs4s/go-cqhttp/global/config.Get(0xc00007c600)
        github.com/Mrs4s/go-cqhttp/global/config/config.go:149 +0x59
main.main()
        github.com/Mrs4s/go-cqhttp/main.go:76 +0x298
```

--- a/global/config/config.go
+++ b/global/config/config.go
@@ -175,6 +175,7 @@ func Get() *Config {
 		global.SetAtDefault(&config.Account.ReLogin.Delay, uint(toInt64(os.Getenv("GCQ_RELOGIN_DELAY"))), uint(0))
 		global.SetAtDefault(&config.Account.ReLogin.MaxTimes, uint(toInt64(os.Getenv("GCQ_RELOGIN_MAX_TIMES"))), uint(0))
 		dbConf := &LevelDBConfig{Enable: global.EnsureBool(os.Getenv("GCQ_LEVELDB"), true)}
+		config.Database = make(map[string]yaml.Node)
 		config.Database["leveldb"] = func() yaml.Node {
 			n := &yaml.Node{}
 			_ = n.Encode(dbConf)

